urdf/actuator.xacro

```diff
@@ -18,47 +18,47 @@
     <xacro:if value="${model_name == 'X5_1'}">
       <xacro:property name="actuator_mass" value="0.315"/>
       <xacro:property name="effort_limit" value="2.5"/>
-      <xacro:property name="velocity_limit" value="90"/>
+      <xacro:property name="velocity_limit" value="9.425"/>
```

Comment:
what are the units here?  I thought the previous values with the rad/s values from the datasheets., but maybe that is not correct?